backend/app/rest/proxy/image.go

```diff
@@ -84,15 +84,6 @@ func (p Image) replace(commentHTML string, imgs []string) string {
 
 // Handler returns http handler respond to proxied request
 func (p Image) Handler(w http.ResponseWriter, r *http.Request) {
-	if !p.HTTP2HTTPS && !p.CacheExternal {
```

Comment:
not 100% sure but I think it effectively enforces http->https proxy regardless the settings

Reply:
Converter won't alter text of new comments. But existing links will continue to work, which is exactly what we want.